pymeasure/instruments/siglenttechnologies/siglent_spd1305x.py

```diff
@@ -29,17 +30,19 @@ class SPD1305X(SPDSingleChannelBase):
     """Represent the Siglent SPD1305X Power Supply.
     """
 
+    voltage_range = [0, 30]
+    current_range = [0, 5]
+    channels = Instrument.ChannelCreator(SPDChannel, 1,
+                                         voltage_range=voltage_range,
+                                         current_range=current_range)
```

Comment:
At first I thought, that it does not work, but a fast test confirmed that it works.
Thanks for showing it, because that way also enums could work!

Reply:
The 1305 class was getting complicated enough that I felt it needed some test visibility. Glad it was useful. Does this mean that there is a more concise or clear way of doing what I've done here, if you didn't think it would work in the first place?

Reply:
This is the best way!

I previously tried make something similar to work, but failed, because I used `SPD1305.voltage_values`, which did not work, because `SPD1305` is not yet defined. Therefore I told you to put the variables at module level (i.e. outside the class).

With all that `self` of the instances, I forgot, that class variables are "local variables" inside the class.

Reply:
Great, thanks